style.css

```diff
@@ -249,20 +257,56 @@ h4 {
 }
 
 .btn:hover {
-  color: #7d73fc;
+  background-color: #7d73fc;
 }
 
 .btn:active {
-  color: #7d73fc;
+  background-color: #e5e5e5;
+}
+
+.btn:enabled {
+  background-color: #7d73fc;
 }
 
 .btn:disabled {
-  color: #7d73fc;
+  background-color: #000;
 }
 
 .email-link {
   color: #fff;
   text-decoration: none;
   text-align: center;
   margin-top: 10px;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 5px;
+}
+
+.btn-1 {
+  width: 10rem;
+  height: 48px;
+  margin-left: 25px;
+  margin-top: 20px;
+  background-color: #fff;
+  color: #fff;
+  font-size: 20px;
+  font-weight: 500;
+  border: 0;
+}
+
+.btn-1:hover {
+  background-color: #7d73fc;
+  box-shadow: 0 8px 16px rgba(125, 115, 252, 0.48);
+}
+
+.btn-1:active {
+  background-color: #5e42d7;
+}
+
+.btn-1:enabled {
+  background-color: #2a2b33;
+}
+
+.btn-1:disabled {
+  background-color: #000;
```

Comment:
- [ ] I think your background color for btn:diabled should remain  `background: #2A2B33;`